CODE-OF-CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at codingclub.iiitv@gmail.com. All
+reported by contacting the project team at admin@cheflib.co. All
```

Comment:
Better put it in code blocks like this 
`admin@cheflib.co`

Reply:
@aashutoshrathi Done. PTAL